README.rst

```diff
@@ -155,6 +157,26 @@ The backend will return a JSON response:
         "S3BotoStorageHealthCheck": "working"
     }
 
+Overriding a health check
```

Comment:
I don't thing this is the way it should be done. Simply don't register them in the first place. We only register the plugins if the apps are added.

I am happy to clarify this better in the docs but it's out of scope for this PR.

Reply:
Please drop the entire section.